SD_Card_Read/SD_Card_Read.ino

```diff
@@ -0,0 +1,56 @@
+#include <SD.h>
+
+const int chipSelectPin = 28;
+
+void setup() {
+  // put your setup code here, to run once:
+
+  Serial.begin(9600);
+
+  while (!Serial) {
+    Serial.print("Initializing SD card...");
+  }
```

Comment:
That's not how this works. `!Serial` is true when the serial connection isn't established yet, so `Serial.print` won't work. Instead, it's common to do
```cpp
while (!Serial) {
  // do nothing until the serial port is ready
}
```